packages/shared/lib/clients/orchestrator.ts

```diff
@@ -218,22 +203,6 @@ export class Orchestrator {
                 }
             });
 
-            await telemetry.log(
-                LogTypes.ACTION_FAILURE,
-                content,
-                LogActionEnum.ACTION,
-                {
-                    error: stringifyError(err),
-                    input: JSON.stringify(input),
-                    environmentId: String(connection.environment_id),
-                    connectionId: connection.connection_id,
-                    providerConfigKey: connection.provider_config_key,
-                    actionName,
-                    level: 'error'
-                },
-                `actionName:${actionName}`
-            );
-
```

Comment:
I think monitors and health dashboard widgets depends on this ACTION_SUCCESS/ACTION_FAILURE logs

Reply:
can be replaced by nango.orch.action.failure and nango.orch.action.success

Reply:
however I'm seeing discrepancy between the two, not sure why especially weird since they are logged at the same place

Reply:
actually that might just be log times that differ

Reply:
changed that widget to use metrics